config/locales/en/citizens.yml

```diff
@@ -102,32 +103,25 @@ en:
         page_heading: What regular payments do you make?
       update:
         none_selected: Select if you make any regular payments
-    information:
-      show:
-        heading: Give one-time access to your bank accounts
-        secure_service_redirect_info: We'll transfer you to a secure service to sign in to your online banking.
-        what_will_need_to_be_shared: You'll need to share all of your bank accounts with us, including any in joint names or with no money.
-        online_banking_info_heading: How we connect to your online banking
-        data_download_info: After you've signed in and authorised us to access your account, we download bank transactions covering the last 3 months.
-        banking_credentials_not_saved: We do not save your online banking username and password.
     legal_aid_applications:
       index:
         case_reference: 'Case reference:'
         heading_1: Share your financial information with us
+        how_we_do_this_heading: How we do this
+        how_we_do_this_body_1: The Legal Aid Agency (LAA) uses a secure third-party service to connect you to your online banking.
```

Comment:
I see that this isn't a list translation. I'm guessing to not make it a bulleted list. Is the reason why it does not have bullet points a designer decision?

Reply:
Yes, the design's here: https://apply-for-legal-aid-prototype.herokuapp.com/apply_for_legal_aid_prototype_non_passported_provider_means/citizen_start_page

It's basically several paragraphs of text rather than a list. I did consider adding it as a single translation item with the html to break it up included, eg
```
<p>The Legal Aid Agency (LAA) uses a secure third-party service to connect you to your online banking.</p><p> Once you're signed in to online banking, we ask you to give us one-time access to your bank accounts.</p>...
```
but it looks pretty ugly and would make it harder to change in future. I think this way is a little clunky but at least it's clear what's going on.

Reply:
Makes sense, thank you.